static/js/topic_page/map_tile.tsx

```diff
@@ -275,11 +279,18 @@ function draw(
   const getTooltipHtml = (place: NamedPlace) => {
     let value = "Data Unavailable";
     if (place.dcid in chartData.dataValues) {
-      value = formatNumber(
-        Math.round((chartData.dataValues[place.dcid] + Number.EPSILON) * 100) /
-          100,
-        props.statVarSpec.unit
-      );
+      // shows upto 4 decimal points for very low values
+      if (chartData.dataValues[place.dcid] < 0.01) {
```

Comment:
let's do this for all values where the absolute value (in case there are negatives) is between 0 and 1

Reply:
That's a great catch! Changed it to absolute value.

Reply:
please also change the check so that this case is only for numbers between 0 and 1